### PR TITLE
[improvement](resource-group) add test for resource group

### PR DESCRIPTION
**Pull request by luozenglin**
# Proposed changes

Issue Number: #18098

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by luozenglin** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 33.26 seconds
 stream load tsv:          437 seconds loaded 74807831229 Bytes, about 163 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          72 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230413015914_clickbench_pr_128288.html

**Comment by luozenglin** [comment]
run buildall

**Comment by luozenglin** [comment]
run buildall

**Comment by luozenglin** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/resource/resourcegroup/ResourceGroupMgr.java [hunk 1]
```diff
@@ -127,8 +127,11 @@ private void checkAndCreateDefaultGroup() {
     }
 
     public void createResourceGroup(CreateResourceGroupStmt stmt) throws DdlException {
-        ResourceGroup resourceGroup = ResourceGroup.createResourceGroup(stmt.getResourceGroupName(),
-                stmt.getProperties());
+        if (!Config.enable_resource_group) {
+            throw new DdlException("Please add `enable_resource_group=true` in fe.conf to enable resource group.");
```
Comment (wangbo) [inline fe/fe-core/src/main/java/org/apache/doris/resource/resourcegroup/ResourceGroupMgr.java:131]:
```suggestion
            throw new DdlException("unsupported feature now,coming soon");
```

**Comment by luozenglin** [comment]
run buildall

**Comment by luozenglin** [comment]
run buildall

**Review by wangbo — APPROVED** [review]
LGTM

**Pull request merged**